Components/input/input.css

```diff
@@ -0,0 +1,37 @@
+@import url('../../globalVar.css');
+.input-container{
+    align-items: flex-start;
+    column-gap: 1rem;
+    display: flex;
```

Comment:
Always write display first. How will align-items and other flex property will work if you write them before display:flex? Not good for debugging purpose.

Reply:
Hi Harshit,
I am following alphabetical wise ordering of css properties, as it was mentioned in one of the CSS lectures